src/lib/mongo.js

```diff
@@ -62,6 +65,8 @@ var initReplSet = function(db, hostIpAndPort, done) {
         return done(err);
       }
 
+      console.log('initial config is', config);
```

Comment:
I think I'd prefer to see the callback function parameter renamed here instead... maybe turn it into `rsConfig` - makes the most sense to me.